tests/test_ddk.py

```diff
@@ -49,6 +69,23 @@ def test_J1713(self):
         log.debug("Max diff %lf" % np.abs(diff - diff.mean()).max())
         assert np.all(np.abs(diff - diff.mean()) < 5e-7), "DDK J1713 TEST FAILED"
 
+    def test_change_px(self):
+        self.modelJ1713.update_binary_object(toas=self.toasJ1713)
+        assert self.modelJ1713.binary_instance.PX.value == self.modelJ1713.PX.value
+        bdelay0 = self.modelJ1713.binary_instance.binary_delay()
+        b_time0 = self.modelJ1713.binary_instance.t
+        # Change PX value
+        self.modelJ1713.PX.value = 0.1
+        self.modelJ1713.update_binary_object()
+        b_time1 = self.modelJ1713.binary_instance.t
+        assert self.modelJ1713.binary_instance.PX.value == 0.1
+        # The stand alone binary model's input time should not change
```

Comment:
Shouldn't the binary model's time variable be barycentered times - which depend on PX for non-barycentered TOAs? Why are these time variables the same?

Reply:
I want to isolate the PX contribution in the barycentering and DDK model. I specifically designed the test this way.

Reply:
Right, but I'm not clear why it works - why do the two `t` variables have the same value once `PX` has been changed?

Reply:
The stand-alone model has a choice that not updating the input time only the parameter. Here the variable t does not get updated.

Reply:
Is that documented anywhere? Could you please add a comment explaining that that's what's going on, and also document the update function so we can tell what's going on? That seems like a potentially surprising behaviour.

Reply:
Will add it.

Reply:
The documentation is added.